zapisy/apps/users/urls.py

```diff
@@ -9,6 +9,8 @@
     path('employees/', views.employees_view, name='employees-list'),
     path('students/', views.students_view, name='students-list'),
     path('employees/<int:user_id>/', views.employees_view, name='employee-profile'),
+    path("employess/<int:user_id>/semester/<int:semester_id>/", views.employees_view, name="employee-profile-semester"),
```

Comment:
Ależ wspaniałe jest to Django, że możemy popełnić taką literówkę a mimo to wszystko działa jak trzeba :slightly_smiling_face: 
```suggestion
    path("employees/<int:user_id>/semester/<int:semester_id>/", views.employees_view, name="employee-profile-semester"),
```
Proszę też w obu dodanych tu liniach zamienić podwójne cudzysłowy na pojedyncze zgodnie z widoczną w dotychczasowej zawartości pliku konwencją.